src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -268,6 +268,17 @@ public void pack_run(ChocolateyConfiguration config)
                 properties["version"] = config.Version;
             }
 
+            // Add any other properties passed to the pack command overriding any present.
+            foreach (var property in config.PackCommand.Properties)
+            {
+                this.Log().Debug(() => "{0} property '{1}': {2}".format_with(

```

Comment:
I see. I spent some time grokking the code base but focused more on areas I needed to update. So would you prefer `Trace` or `Debug`?